specmatic.json

```diff
@@ -0,0 +1,17 @@
+{
+  "sources": [
+    {
+      "provider": "",
+      "test": [],
+      "stub": []
+    }
+  ],
+  "environments": {
+    "local": {
+      "baseurls": {},
+      "variables": {
+        "key": "updated-value"
```

Comment:
initial-value is a more apt for what this signifies

Reply:
Added changes